pvlib/tests/test_modelchain.py

```diff
@@ -2063,3 +2063,40 @@ def test__irrad_for_celltemp():
     assert len(poa) == 2
     assert_series_equal(poa[0], effect_irrad)
     assert_series_equal(poa[1], effect_irrad)
+
+
+@pytest.mark.parametrize('strategy, strategy_str', [
+    ('south_at_latitude_tilt', 'south_at_latitude_tilt'),
+    (None, 'None')])  # GitHub issue 352
```

Comment:
Thanks for putting this test back.  I think this decorator is no longer needed and can be deleted.